website/src/docs/contributing.md

```diff
@@ -26,7 +26,10 @@ Unit tests are using Jest and can be run with:
 
 For acceptance (or end to end) tests, we use [Webdriverio](http://webdriver.io). For it to run locally, you need to install selenium standalone server, just follow [the guide](http://webdriver.io/guide.html) to do so. You can also install Selenium Standalone server from NPM:
 
-`npm install selenium-standalone`
+```bash
+npm install selenium-standalone
+selenium-standalone install
+```
```

Comment:
I wonder if it’s better to do `npm install selenium-standalone -g` as suggested on https://www.npmjs.com/package/selenium-standalone

Reply:
Yes, I forgot to update that part. I had to use the `-g` flag or it wouldn't install, well noted :) Will update the PR.

Reply:
Done